tmux-thumbs.tmux

```diff
@@ -2,12 +2,15 @@
 
 CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-DEFAULT_THUMBS_KEY=space
+DEFAULT_THUMBS_KEY_OPTIONS=""
+THUMBS_KEY_OPTIONS="$(tmux show-option -gqv @thumbs-key-options)"
+THUMBS_KEY_OPTIONS=${THUMBS_KEY_OPTIONS:-$DEFAULT_THUMBS_KEY_OPTIONS}
 
+DEFAULT_THUMBS_KEY=space
 THUMBS_KEY="$(tmux show-option -gqv @thumbs-key)"
 THUMBS_KEY=${THUMBS_KEY:-$DEFAULT_THUMBS_KEY}
 
-tmux bind-key "${THUMBS_KEY}" run-shell -b "${CURRENT_DIR}/tmux-thumbs.sh"
+tmux bind-key "${THUMBS_KEY_OPTIONS}" "${THUMBS_KEY}" run-shell -b "${CURRENT_DIR}/tmux-thumbs.sh"
```

Comment:
This would send an empty (zero length string) argument to tmux by default.  A small test shows tmux doesn't like it:

```sh
$ tmux bind-key "" "X" display-message hello
unknown key:
exit 1

$ p tmux bind-key "" "X" display-message hello
["tmux", "bind-key", "", "X", "display-message", "hello"]
```

Essentially, you need to avoid sending the empty argument to tmux by default.  Relaxing the quoting discipline is one way to solve this.  But since you wish to maintain the quoting discipline, an explicit if-statement would also solve this:

```sh
if [ -n "${THUMBS_KEY_OPTIONS}" ]; then
  tmux bind-key "${THUMBS_KEY_OPTIONS}" "${THUMBS_KEY}" run-shell -b "${CURRENT_DIR}/tmux-thumbs.sh"
else
  tmux bind-key "${THUMBS_KEY}" run-shell -b "${CURRENT_DIR}/tmux-thumbs.sh"
fi
```

Reply:
Ok, got it. Although I'm going to try another approach because this last if is duplicating a bit the logic.

Reply:
Can you try again with the latest code?